packages/devices/origin-device-registry-irec-local-api/src/device/device.controller.ts

```diff
@@ -76,7 +76,7 @@ export class DeviceController {
         description: 'Returns all IREC fuels'
     })
     getFuels(): CodeNameDTO[] {
-        const deviceTypes = this.deviceService.getDeviceTypes();
+        const deviceTypes = this.deviceService.getFuelTypes();
```

Comment:
I think it is a mismatch. `this.deviceService.getDeviceTypes()` should be used here

Reply:
done